src/main/java/org/sagebionetworks/bridge/models/surveys/SurveySQL.java

```diff
@@ -0,0 +1,86 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import java.util.List;
+
+import org.sagebionetworks.bridge.hibernate.HibernateSurvey;
+import org.sagebionetworks.bridge.hibernate.HibernateSurveyElement;
+import org.sagebionetworks.bridge.hibernate.HibernateSurveyQuestion;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+import org.sagebionetworks.bridge.models.GuidCreatedOnVersionHolder;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.collect.ImmutableList;
+
+@BridgeTypeName("Survey")
+@JsonDeserialize(as=HibernateSurvey.class)
+public interface SurveySQL extends GuidCreatedOnVersionHolder, BridgeEntity  {
```

Comment:
Similarly, this class is also nearly identical to Survey. Why not use the same interface for both?

Reply:
I can't have two create methods to create both DynamoSurvey() and HibernateSurvey() so I thought I'd need a new interface for this?

Reply:
When we call create(), it's because we don't care what the underlying implementation is, so it doesn't matter whether we return a DynamoSurvey or a HibernateSurvey. Similarly, when we deserialize from JSON.

If we get into a situation where we _do_ care (such as the DynamoSurveyDao needing to save a survey to Dynamo), we could use the copy constructors to convert them. (In particular, it looks like SurveyElementFactory.fromDynamoEntity() doesn't create things _from_ DynamoSurveyElement, but rather it converts things _to_ DynamoSurveyElement.)